core/src/main/java/nu/yona/server/goals/service/GoalDTO.java

```diff
@@ -45,6 +48,11 @@ public UUID getID()
 		return id;
 	}
 
+	public ZonedDateTime getCreationTime()
```

Comment:
To change the serialization format, you will need to add an annotation. In ``AppActivityDTO``, I used the following for deserialization:
```
@JsonFormat(pattern = **"yyyy-MM-dd'T'HH:mm:ss.SSSZ")
```